functions/load_data_utils.py

```diff
@@ -35,33 +43,33 @@ def extract_relevant_value(d):
 
     return dict_df
 
-def get_specialty_group(l):
 
+def get_specialty_group(x):
     if (
-        l.count("Medicine") > 0
-        or l.count("Geriatric")
-        or l.count("Cardiology")
-        or l.count("Pharmacology ") > 0
-        or l.count("Pharmacy ") > 0
-        or l.count("Endocrinology") > 0
-        or l.count("Rheumatology") > 0
-        or l.count("Gastroenterology") > 0
-        or l.count("Infectious Diseases") > 0
-        or l.count("General Practice") > 0
-        or l.count("Dermatology") > 0
+        x.count("Medicine") > 0
+        or x.count("Geriatric")
+        or x.count("Cardiology")
+        or x.count("Pharmacology ") > 0
+        or x.count("Pharmacy ") > 0
+        or x.count("Endocrinology") > 0
+        or x.count("Rheumatology") > 0
+        or x.count("Gastroenterology") > 0
+        or x.count("Infectious Diseases") > 0
+        or x.count("General Practice") > 0
+        or x.count("Dermatology") > 0
     ):
         return "medical"
     elif (
-        l.count("Surgery") > 0
-        or l.count("Anaesthetics") > 0
-        or l.count("Maternity") > 0
-        or l.count("Obstetrics") > 0
-        or l.count("Orthopaedics") > 0
-        or l.count("Otolaryngology") > 0
-        or l.count("Urology") > 0
-        or l.count("Dental") > 0
+        x.count("Surgery") > 0
+        or x.count("Anaesthetics") > 0
+        or x.count("Maternity") > 0
+        or x.count("Obstetrics") > 0
+        or x.count("Orthopaedics") > 0
+        or x.count("Otolaryngology") > 0
+        or x.count("Urology") > 0
+        or x.count("Dental") > 0
     ):
         return "surgical"
-    elif l.count("Oncology") > 0 or l.count("Haematology") > 0:
+    elif x.count("Oncology") > 0 or x.count("Haematology") > 0:
         return "haem_onc"
-    return "medical"
\ No newline at end of file
+    return "medical"
```

Comment:
Ruff did not like the use of a single character `l` as a variable name as it can be hard to distinguish from `1` in some fonts. I wasn't sure what would be a better longer name here so I just swapped for a generic `x` but something more descriptive would be better.